WooCommerce/Classes/ViewRelated/Products/Variations/ProductsTabProductViewModel+ProductVariation.swift

```diff
@@ -2,42 +2,50 @@ import UIKit
 import Yosemite
 
 extension ProductsTabProductViewModel {
-    init(productVariationModel: EditableProductVariationModel, currency: String) {
+    init(productVariationModel: EditableProductVariationModel, currency: String, currencySettings: CurrencySettings = .shared) {
```

Comment:
ℹ️  Since we now pass the `CurrencySettings`, could we perhaps infer the value of `currency` using the `CurrencySettings` object?

```swift
let currencyString = currencySettings.symbol(from: currencySettings.currencyCode)
```

What am I missing? 😅

Reply:
Great catch! `currency` isn't necessary anymore since it can be derived from `CurrencySettings`. updated here: https://github.com/woocommerce/woocommerce-ios/pull/2627/commits/a8c5014a07edc0de5f4075bd2b14d8950bbdef64

Reply:
Yay!